metrics/character/tests.py

```diff
@@ -0,0 +1,35 @@
+from statistics import mean, median, stdev
```

Comment:
@lvwerra, do you have a suggestion for where tests like these should go? We don't currently keep metric-specific tests in the metric folders, but these are handy to have around. (@BramVanroy FYI Leandro is on vacation until next week!)

Reply:
If I remember correctly, the `tests.py` is created automatically when using `evaluate-cli create`.

Reply:
Hi @BramVanroy and @mathemakitten, the `tests.py` is WIP and currently does nothing. The idea was originally that we could setup a list of example inputs/outputs in that file that we could then use to test the metric as well as use them as examples in the Gradio app. 

The metrics are currently tested with doctest. That means the examples in the metric's docstring are executed in the tests. So if you think there are important edge cases to be tested you could add them there.